cro-clib/chain.h

```diff
@@ -19,6 +19,8 @@ typedef struct CroJsonRpc CroJsonRpc;
 
 typedef struct CroTx CroTx;
 
+typedef struct Option_ProgressCallback Option_ProgressCallback;
```

Comment:
where is Option_ProgressCallback?
opaque pointer?

Reply:
how about "OptionProgressCallback"?
removing _

Reply:
This is auto-generated by cbindgen from `Option<ProgressCallback>`.

Reply:
I guess the example needs updating: https://github.com/crypto-com/chain/blob/master/cro-clib/examples/rpc.c#L12 ?

Reply:
or if one passes non-null function pointer, it'll work as intended / before?

Reply:
> or if one passes non-null function pointer, it'll work as intended / before?

yes, the null pointer will be treated as a `None`.